src/components/Users.js

```diff
@@ -1,14 +1,71 @@
 import React from 'react'
+import users from '../data/users.json'
+import { Grid } from 'semantic-ui-react'
+import ReactTable from 'react-table'
+import { filteredUsers } from '../helpers/filterUsers'
+import AddUsers from '../containers/AddUsers'
+import { addDataToDisplay } from '../helpers/addDataToDisplay'
 
 
 class Users extends React.Component {
+
+  state = {
+      data: users,
+      search: ''
+    }
+
   render() {
+    let data = this.state.data
```

Comment:
Jeżeli nie mutujesz zmiennych, to lepiej użyć `const. Dodatkowo, lepiej jest użyć jednej deklaracji, zamiast kilku, tzn. lepsze to:
```
const foo = 42,
  bar = 'wat'
```
niż
```
const foo = 42
const bar = 'wat'
```